applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -168,3 +170,15 @@ def Initialize(self):
 
     def AdvanceInTime(self, current_time):
         raise Exception("AdvanceInTime is not implemented. Potential Flow simulations are steady state.")
+
+    def FinalizeSolutionStep(self):
+        (self.solver).FinalizeSolutionStep()
+
+    def SolveSolutionStep(self):
```

Comment:
I think it should be possible not to reimplement SolveSolutionStep and FinalizeSolutionStep. You might need to define self.min_buffer_size in your solver

Reply:
I set the buffer_size inside ale_potential_flow file as 1 and erased FinalizeSolutionStep() and SolveSolutionStep() from potential_flow_solver and it works perfectly fine.